src/trace/trace.h

```diff
@@ -0,0 +1,49 @@
+/* ==========================================================================
+ * trace.h - support generic tracing layers.
+ * --------------------------------------------------------------------------
+ * Copyright (C) 2025  zhenwei pi <pizhenwei@bytedance.com>
+ * Copyright (C) 2025  zhiqiang li <lizhiqiang.sf@bytedance.com>
+ *
+ * This work is licensed under BSD 3-Clause, License 1 of the COPYING file in
+ * the top-level directory.
+ * ==========================================================================
+ */
+/*
+ * Copyright (c) Valkey Contributors
+ * All rights reserved.
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+#if !defined(__VALKEY_TRACE_H__)
+#define __VALKEY_TRACE_H__
+
+#include "trace_db.h"
+#include "trace_cluster.h"
+#include "trace_sys.h"
+#include "trace_aof.h"
+#include "trace_server.h"
+#include "trace_commands.h"
+
+typedef struct valkeyTraceEvents {
+    unsigned aof : 1;
+    unsigned server : 1;
+    unsigned cluster : 1;
+    unsigned sys : 1;
+    unsigned db : 1;
+    unsigned commands : 1;
+    unsigned enabled : 1;
+    unsigned reserved : 1;
```

Comment:
remove 'unsigned reserved : 1;' would be fine

Reply:
typedef struct valkeyTraceEvents {
   union struct {
    unsigned aof : 1;
    unsigned server : 1;
    unsigned cluster : 1;
    unsigned sys : 1;
    unsigned db : 1;
    unsigned commands : 1;
   };
    unsigned enabled;
};

if any bit is specified, enabled would not 0.
do you like this style?